src/main/java/pl/debuguj/parkingspacessystem/dao/ParkingSpaceDaoMockImpl.java

```diff
@@ -0,0 +1,117 @@
+package pl.debuguj.parkingspacessystem.dao;
+
+import org.joda.time.DateTime;
+import org.springframework.stereotype.Service;
+import pl.debuguj.parkingspacessystem.domain.ParkingSpace;
+import pl.debuguj.parkingspacessystem.exceptions.CarRegisteredInSystemException;
+import pl.debuguj.parkingspacessystem.exceptions.IncorrectEndDateException;
+import pl.debuguj.parkingspacessystem.exceptions.ParkingSpaceNotFoundException;
+
+import java.util.*;
+import java.util.stream.Collectors;
+
+/**
+ * Created by grzesiek on 07.10.17.
+ */
+@Service
+public class ParkingSpaceDaoMockImpl implements ParkingSpaceDao {
```

Comment:
to nie musi być publiczna klasa. To nie jest `Mock`.

Reply:
Chyba wszystkie klasy w tym projekcie mogą być prywatne. Nie zwróciłęm na to uwagi. Stosowanie nazwy "Mock" w dao to mi zostało po jednym z kursów, być może błędnie.

Reply:
Chodziło mi o to, że są różne rodzaje tzw "Mocków": Stub, Mock, Fake, (Spy). Fajny wpis jest tutaj: https://blog.pragmatists.com/test-doubles-fakes-mocks-and-stubs-1a7491dfa3da